src/sentry/event_manager.py

```diff
@@ -1136,6 +1136,30 @@ def _save_aggregate(self, event, hashes, release, **kwargs):
                 event.event_id,
             )
 
+        # Check if already assigned
+        # TODO: convert to model method
+        from sentry.models import GroupAssignee
+        assignee = GroupAssignee.objects.filter(group_id=group.id).first()
+        logger.info('-- starting auto-assignment process --')
```

Comment:
log statements need to be thought of as singular events vs a text log of print statements